packages/edit-site/src/components/sidebar-edit-mode/page-panels/edit-template.js

```diff
@@ -53,15 +56,23 @@ export default function EditTemplate() {
 				isTemplateHidden:
 					getCanvasMode() === 'edit' &&
 					getPageContentFocusType() === 'hideTemplate',
+				postType: _postType,
 			};
 		},
 		[]
 	);
 
+	const [ blocks ] = useEntityBlockEditor( 'postType', postType );
```

Comment:
Because it's an edge case, I'm happy to remove this if it's not performant, especially if the blockList is large.

Reply:
From my perspective, I think this is probably okay, as we're only rendering it when we're in `hasPageContentFocus` so the performance impact is limited to the page editing experience. It also wouldn't be hard to revert if it does wind up causing any performance issues.